### ISSUE TITLE
Unusable shell prompt after installation

### ISSUE THREAD

**Issue by thorstenkampe** [oh-my-fish/oh-my-fish#541]
The oh-my-fish installation sets the default theme after installation which uses the right prompt to display the clock (probably if there is no right prompt defined). This leads to an unusable shell on Windows with ConEmu and on Linux with GNOME Terminal (see https://github.com/fish-shell/fish-shell/issues/4306).

The underlying issue here is, I think, that the oh-my-fish installer should not enable any kind of theme until specifically asked (when a user installs a theme and oh-my-fish enables it, it is fine).
![image](https://user-images.githubusercontent.com/475462/29221545-7848a6da-7ebf-11e7-904d-37cb4ee433ed.png)

**Comment by sagebind** [comment]
Looks very similar to #534

**Comment by oranja** [comment]
It sucks, but these things happen.
Since this is a severe `fish` bug that can occur in many other configurations - not just with OMF's default theme - I'd rather wait and let it play out.

Unless `fish` decides to retire all right prompts, or announces that it's going to take a long while to fix, I think `default` should stay as a theme. Having the theme change immediately when a user just tries `fish` and `omf` for the first time is exciting and one of OMF's "marketing" to make the user experiment with other themes and even publishing her own. To this day, seeing the `default` theme after just setting up `fish` and `omf` on a new setup always gives me the warm feeling of arriving home.

With that being said, what do you guys think on adding a PSA to the top of the README that describes the problem and suggests a simple fix? (something that can be done from `bash` or a file manager)

**Comment by sagebind** [comment]
I'd propose we add a "Known issues" piece to the docs and list it there. Even though it is not our issue, it might be the first place users look when they encounter some sort of issue like this.

**Comment by sagebind** [comment]
This is not our issue, and we've now added a note in the README concerning this Fish bug.

**Comment by gerardbosch** [comment]
Hi there! I think the "Known issues" in the README should be more visible, maybe at the begining. I understand it is not your issue but as a new user of fish/omf, I would like to notice the issue to be aware and also understand how to fix it. Maybe a note on how to fix/workaround it with another theme would be appreciated. Thanks :)

P.S. Does a downgrade of fish would solve the problem?

**Comment by gerardbosch** [comment]
From README:

>Due to a regression bug in fish 2.6 with some terminal emulators, right prompts make the shell unusable. OMF's default theme features a right prompt, so it's necessary to use an alternative theme until a fix is released. (see #541)

I am using MSYS2 + Cmder and tried downgrading gradually fish until fish 2.1.2 and did not solved the problem :(

**Comment by AnderssonPeter** [comment]
Is this bug still present in the 2.7.1 version of fish, if not please update the Readme file with that information!

**Comment by sagebind** [comment]
Unfortunately I don't think anyone on the core team here use Windows so we are unable to test it. 

Also, 

> tried downgrading gradually fish until fish 2.1.2

Oh My Fish supports only Fish 2.2.0 or newer.

**Comment by Natetronn** [comment]
A couple years later but, I ran into this as well on Manjaro. Maybe it will help others.

What I did to fix it was, I uninstalled oh-my-fish and the [Tide theme](https://github.com/IlanCosman/tide) as well. Then I installed oh-my-fish (first this time) and finally I installed Tide theme again after. Seems to have address the issue and my prompt is in the correct location.

oh-my-fish Version 7
Tide v3.2.0
